### PR TITLE
Add parameter tflint_config

### LINKED ISSUE #57: No way to pass flags to `tflint --init`
Use case: I have a repository with a single `.tflint.hcl` which contains the `terraform` and `aws` plugins. I'd like this file to be the single source of truth for how `tflint` is configured to run both locally and in CI.

I can almost achieve this by using the `flags` argument and passing `--config` to `tflint` directly:

```
- uses: reviewdog/action-tflint@v1.17.0
  with:
    tflint_init: true
    flags: --config=${{ github.workspace }}/terraform/.tflint.hcl
```

However, since the value of the `flags` argument is [never passed](https://github.com/reviewdog/action-tflint/blob/46e609666b039b775a150e84781ef79ea90089a8/script.sh#L76) to the `tflint --init` command, the plugins are never installed, resulting in the following error:

```
Running tflint with reviewdog 🐶 ...
  Failed to initialize plugins; Plugin `terraform` not found. Did you run `tflint --init`?
```

### PR DESCRIPTION
**Pull request by thaim**
fixes #57

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -73,7 +73,7 @@ done
 case "${INPUT_TFLINT_INIT:-false}" in
     true)
         echo "::group:: Initialize tflint from local configuration"
-        TFLINT_PLUGIN_DIR="${TFLINT_PLUGIN_DIR}" GITHUB_TOKEN="${INPUT_GITHUB_TOKEN}" "${TFLINT_PATH}/tflint" --init
+        TFLINT_PLUGIN_DIR="${TFLINT_PLUGIN_DIR}" GITHUB_TOKEN="${INPUT_GITHUB_TOKEN}" "${TFLINT_PATH}/tflint" --init -c ${INPUT_TFLINT_CONFIG}
```
Comment (shogo82148) [inline script.sh]:
```suggestion
        TFLINT_PLUGIN_DIR="${TFLINT_PLUGIN_DIR}" GITHUB_TOKEN="${INPUT_GITHUB_TOKEN}" "${TFLINT_PATH}/tflint" --init -c "${INPUT_TFLINT_CONFIG}"
```

File: script.sh [hunk 2]
```diff
@@ -87,7 +87,7 @@ case "${INPUT_TFLINT_INIT:-false}" in
 esac
 
 echo "::group:: Print tflint details ..."
-  "${TFLINT_PATH}/tflint" --version
+  "${TFLINT_PATH}/tflint" --version -c ${INPUT_TFLINT_CONFIG}
```
Comment (shogo82148) [inline script.sh]:
```suggestion
  "${TFLINT_PATH}/tflint" --version -c "${INPUT_TFLINT_CONFIG}"
```

File: script.sh [hunk 3]
```diff
@@ -98,7 +98,7 @@ echo '::group:: Running tflint with reviewdog 🐶 ...'
   set +Eeuo pipefail
 
   # shellcheck disable=SC2086
-  TFLINT_PLUGIN_DIR=${TFLINT_PLUGIN_DIR} "${TFLINT_PATH}/tflint" --format=checkstyle ${INPUT_FLAGS} ${INPUT_TFLINT_TARGET_DIR} \
+  TFLINT_PLUGIN_DIR=${TFLINT_PLUGIN_DIR} "${TFLINT_PATH}/tflint"  -c ${INPUT_TFLINT_CONFIG} --format=checkstyle ${INPUT_FLAGS} ${INPUT_TFLINT_TARGET_DIR} \
```
Comment (shogo82148) [inline script.sh]:
```suggestion
  TFLINT_PLUGIN_DIR=${TFLINT_PLUGIN_DIR} "${TFLINT_PATH}/tflint" -c "${INPUT_TFLINT_CONFIG}" --format=checkstyle ${INPUT_FLAGS} ${INPUT_TFLINT_TARGET_DIR} \
```

**Review by shogo82148 — APPROVED** [review]

**Comment by review-dog** [comment]
Hi, @thaim! We merged your PR to reviewdog! 🐶
Thank you for your contribution! ✨

**We just invited you to join the @reviewdog organization on GitHub.**
Accept the invite by visiting https://github.com/orgs/reviewdog/invitation.
By joining the team, you'll be a part of reviewdog community and can help the maintenance of reviewdog.

Thanks again!

**Pull request merged**